packages/quantic/force-app/main/default/lwc/quanticResultCopyToClipboard/quanticResultCopyToClipboard.js

```diff
@@ -107,6 +108,7 @@ export default class QuanticResultCopyToClipboard extends LightningElement {
         this.engine.dispatch(this.actions.logCopyToClipboard(this.result));
         this.displayedLabel = this.successLabel;
         this.resetOriginalLabel();
+        this.template.host.focus();
```

Comment:
This sets focus after the fallback? Because the fallback uses the create element thing to focus, then copy?

Might want to add a comment specifically explaining that right there so we understand in 2 weeks what this does ;)

Reply:
sure yeah👌